pokemons/tests/test_helpers.py

```diff
@@ -64,18 +50,31 @@ def test_pokemon_id_outside_api_constraints_breaks(self):
         is_within_limit = 1 <= self.creator_pokemon.id <= 802
         self.assertFalse(is_within_limit)
 
+    @responses.activate
     def test_request_response_is_a_dict(self):
-        response = self.example_request()
+        url = '{pokeapi}/pokemon/{pokemon_id}'.format(
+            pokeapi=settings.POKEAPI_URL,
+            pokemon_id=self.creator_pokemon.id
+        )
+        responses.add(responses.GET, url, status=200, headers={'Content-Type': 'application/json'})
+        response = requests.get(url)
         self.assertEqual(response.status_code, 200)
```

Comment:
Since this can be kinda repetitive, you can use it inside a base `TestCase` class, and then use it to avoid the need to repeat the `status_code` call.